google/resource_google_organization_policy.go

```diff
@@ -99,6 +99,20 @@ var schemaOrganizationPolicy = map[string]*schema.Schema{
 		Type:     schema.TypeString,
 		Computed: true,
 	},
+	"restore_policy": {
+		Type:          schema.TypeList,
+		Optional:      true,
+		MaxItems:      1,
+		ConflictsWith: []string{"boolean_policy", "list_policy"},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"default": {
+					Type:     schema.TypeBool,
+					Required: true,
```

Comment:
I'm a bit confused about the structure of this. The API requires an empty object, so `default` is just a construct here- what made you decide to go for a nested object with a boolean that has to be set to true, instead of just top-leveling that boolean (i.e. `restore_default = true`)?

Reply:
only because in the requirements give me the next structure:
    restore_policy {
        default = true
    }

Reply:
@morgante, this was your idea I believe. What do you think?

Reply:
The reason I specified it that way was for equivalence with the other constraint types, which use a nested structure. `boolean_policy`, for example, also only has a single boolean value but is also nested.

Reply:
Yeah, it makes sense. It's a little different in this case because the others are mirroring the API exactly as it is, whereas this one is adding a field into an object, even though that field doesn't exist. I do think it's probably less confusing than just requiring an empty object though, and a bit more future proof anyway, so sure. I'm down.